channels-rke2.yaml

```diff
@@ -1715,19 +1762,66 @@ releases:
         repo: rancher-rke2-charts
         version: 1.24.008
     featureVersions: *featureVersions-v1
-  - version: v1.28.6+rke2r1
+  - version: v1.27.11+rke2r1
+    minChannelServerVersion: v2.8.0-alpha1
+    maxChannelServerVersion: v2.8.99
+    serverArgs: &serverArgs-v1-27-11-rke2r1
+      <<: *serverArgs-v1-27-7-rke2r2
+      cni:
+        default: calico
+        type: array
+        options:
+        - canal
+        - cilium
+        - calico
+        - flannel
+        - multus,canal
+        - multus,cilium
+        - multus,calico
+    agentArgs: *agentArgs-v1-25-15-rke2r2
+    charts: &charts-v1-27-11-rke2r1
+      <<: *charts-v1-27-10-rke2r1
+      rke2-cilium:
+        repo: rancher-rke2-charts
+        version: 1.15.100
+      rancher-vsphere-csi:
+        repo: rancher-charts
+        version: 3.0.1-rancher101
+      rke2-calico:
+        repo: rancher-rke2-charts
+        version: v3.27.002
+      rke2-calico-crd:
+        repo: rancher-rke2-charts
+        version: v3.27.002
+      rke2-canal:
+        repo: rancher-rke2-charts
+        version: v3.27.0-build2024020601
+      rke2-coredns:
+        repo: rancher-rke2-charts
+        version: 1.29.001
+      rke2-multus:
+        repo: rancher-rke2-charts
+        version: v4.0.2-build2024020800
+      rke2-metrics-server:
+        repo: rancher-rke2-charts
+        version: 2.11.100-build2023051513
+      harvester-csi-driver:
+        repo: rancher-rke2-charts
+        version: 0.1.1700
+    featureVersions: *featureVersions-v1
+  - version: v1.28.7+rke2r1
     minChannelServerVersion: v2.8.3-alpha1
     maxChannelServerVersion: v2.8.99
-    serverArgs: &serverArgs-v1-28-6-rke2r1
+    serverArgs: &serverArgs-v1-28-7-rke2r1
       <<: *serverArgs-v1-27-7-rke2r2
       embedded-registry:
         type: boolean
-    agentArgs: &agentArgs-v1-28-6-rke2r1
+    agentArgs: &agentArgs-v1-28-7-rke2r1
       <<: *agentArgs-v1-25-15-rke2r2
       default-runtime:
         type: string
       disable-default-registry-endpoint:
         type: boolean
-    charts: &charts-v1-28-6-rke2r1
+    charts: &charts-v1-28-7-rke2r1
       <<: *charts-v1-27-10-rke2r1
     featureVersions: *featureVersions-v1
```

Comment:
Follow-up to now-resolved https://github.com/rancher/kontainer-driver-metadata/pull/1354#discussion_r1517912525

Certainly need RKE2 team input (@briandowns) on this but I gather (again, not sure below is correct until we get input from RKE2 team):
1. For `serverargs` we will be missing updated `cni` options that are supported in 1.28.7 same as in v2.7.11. So we _should_ be able to just change anchor to `*serverArgs-v1-27-11-rke2r1`.
2. `agentargs` LGTM
3. `charts` likely will be the same as on 1.27.11 as well so change anchor to `*charts-v1-27-11-rke2r1`
4. `featureVersions` LGTM
5. Not sure if there are any additional args/charts on 1.28.6 -> 1.28.7

So, something like:
```
  - version: v1.28.7+rke2r1
    minChannelServerVersion: v2.8.3-alpha1
    maxChannelServerVersion: v2.8.99
    serverArgs: &serverArgs-v1-28-7-rke2r1
      <<: *serverArgs-v1-27-11-rke2r1
      embedded-registry:
        type: boolean
    agentArgs: &agentArgs-v1-28-7-rke2r1
      <<: *agentArgs-v1-25-15-rke2r2
      default-runtime:
        type: string
      disable-default-registry-endpoint:
        type: boolean
    charts: &charts-v1-28-7-rke2r1
      <<: *charts-v1-27-11-rke2r1
    featureVersions: *featureVersions-v1
```

Reply:
Server args referencing v1.27.11 is fine. The charts ref should be fine as well.

CNI addition to server args:

```yaml
      cni:
        default: calico
        type: array
        options:
        - canal
        - cilium
        - calico
        - flannel
        - multus,canal
        - multus,cilium
        - multus,calico
```

Reply:
@briandowns , just confirming you're fine with the approach of just referencing `serverArgs-v1-27-11-rke2r1` and just adding `embedded-registry` to it. The reason I'm a bit confused as you stated "CNI addition to server args:" with YAML following it but if we reference `serverArgs-v1-27-11-rke2r1` we will get CNI addition taken care of it I believe. Or am I missing something?

Reply:
My fault, it'll get brought in from that tag.